v1_push_api.json

```diff
@@ -0,0 +1,348 @@
+{
```

Comment:
Ah, at the least it should be used for data validation in the endpoint. I don't see a reason to not expose it otherwise.